### PR TITLE
docs(readme):  automated-lighthouse-check.com - paid / tiered

### PR DESCRIPTION
**Pull request by adamhenson**
**Summary**

Adds a bullet to the `Paid / Tiered` section of `readme` for [Automated Lighthouse Check](https://www.automated-lighthouse-check.com/).

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -291,6 +291,8 @@ This section details services that have integrated Lighthouse data. If you're wo
 
 ### Paid / Tiered
 
+* **[Automated Lighthouse Check](https://www.automated-lighthouse-check.com/)** - Lighthouse as a service offering free and premium plans. Monitoring and historical reporting of Lighthouse audits. Provides Slack, CircleCI implementation, PR comment reporting, and other "DevOps" solutions with supporting documentation.
```
Comment (patrickhulce) [inline readme.md]:
```suggestion
* **[Automated Lighthouse Check](https://www.automated-lighthouse-check.com/)** - Lighthouse-as-a-service offering free and premium plans. Provides monitoring and historical reporting of Lighthouse audits with Slack, CircleCI, GitHub, and many more integrations with supporting documentation.
```

some optional edits for you to consider :)

**Comment by adamhenson** [comment]
Right you are @patrickhulce! Thanks for your review. I removed the bullet for Foo and took you up on the suggestion of phrasing change.

**Comment by adamhenson** [comment]
@patrickhulce - I updated the copy further after using your edits. I think it's important to include some of the extra features people might be looking for as they glance through the bullets. I think the blurb is fairly trimmed and inline with others, but let me know if you have anything else you'd like to see updated. I appreciate this project, and all your work on it among the other contributors 👏

**Comment by adamhenson** [comment]
Awesome thanks @patrickhulce!

**Pull request merged**